src/pages/authentication/signup/Signup.tsx

```diff
@@ -15,7 +15,6 @@ import { AppDispatch } from "../../../app/store";
 import { toast } from "react-toastify";
 import { unwrapResult } from "@reduxjs/toolkit";
```

Comment:
What is this used for??

Reply:
for unwrapping the result and basically it is used for error handling and also checking the result after the dispatch .